shell/platform/android/io/flutter/embedding/engine/android/FlutterView.java

```diff
@@ -35,9 +38,18 @@
 public class FlutterView extends FrameLayout {
   private static final String TAG = "FlutterView";
 
+  // View configuration
```

Comment:
These comments don't really explain why these three fields are useful.  I suggest providing a link to some relevant context that explains how these are used.

Reply:
In this section, I suggest pointing to the RenderMode type, and point from there to either Texture or Surface mode, so the user can read about why they might want either.

Reply:
I updated the comments to be more useful.  I didn't forward anything because anyone who is reading this code is likely to just CMD+B their way to the definition they care about.  These comments are pretty much just for implementers to organize fields a little bit.